src/components/converter/converter.test.js

```diff
@@ -54,5 +54,30 @@ describe('Converter', () => {
     wrapper.update();
     expect(wrapper.find('.converted-price').at(0)).to.have.text('246.00 USD');
   });
-});
 
+
+  it('shold set max. amount', () => {
+    const storeWithCurrency = fakeStore({
+      settings: { currency: 'USD' },
+      settingsUpdated: () => {},
+    });
+
+    const props = {
+      t: key => key,
+      value: '',
+      error: false,
+      currency: 'USD',
+      onSetMaxAmount: sinon.spy(),
+    };
+
+    wrapper = mountWithContext(
+      <Converter {...props} store={storeWithCurrency}/>,
+      { storeState: storeWithCurrency },
+    );
+
+    wrapper.find('.set-max-amount').simulate('click');
+    /* eslint-disable no-unused-expressions */
+    expect(props.onSetMaxAmount).to.have.been.calledOnce;
+    /* eslint-enable no-unused-expressions */
```

Comment:
You can use just one comment before the statement 
```
    // eslint-disable-next no-unused-expressions
```
instead of one comment before and one after